openedx/core/lib/celery/__init__.py

```diff
@@ -19,6 +19,15 @@
 
 from celery import Celery
 
+# TEMP: This code will be removed by ARCH-BOM on 4/22/24
```

Comment:
The end date of logging should be updated or removed if we're planning to keep it undefinitely.